package.json

```diff
@@ -33,6 +33,7 @@
     "@nestjs/platform-express": "^8.0.0",
     "@nestjs/swagger": "^5.0.9",
     "bcrypt": "^5.0.1",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
This package is unnecessary, functionality is already part of the @nestjs/jwt package. See auth.service.ts: https://github.com/SciCatProject/scicat-backend-next/blob/0382a0ab14745a9a20154e6b96c2d90b27a64492/src/auth/auth.service.ts#L39